controllers/nnf_workflow_controller.go

```diff
@@ -719,11 +719,7 @@ func (r *NnfWorkflowReconciler) createNnfStorage(ctx context.Context, workflow *
 				if dwArgs["type"] == "lustre" {
 					nnfAllocSet.NnfStorageLustreSpec.TargetType = strings.ToUpper(s.Spec.AllocationSets[i].Label)
 					nnfAllocSet.NnfStorageLustreSpec.BackFs = "zfs"
-					charsWanted := 8
-					if len(dwArgs["name"]) < charsWanted {
-						charsWanted = len(dwArgs["name"])
-					}
-					nnfAllocSet.NnfStorageLustreSpec.FileSystemName = dwArgs["name"][:charsWanted]
```

Comment:
Should there be a generic "Name" field that still records the `dwsArgs["name"]` value? Just wondering for debug it might be handy. Only asking - I haven't really digested this layer yet to fully comprehend it.

Reply:
It wouldn't be used for anything, and we already have a way to tie the nnfStorage back to which #DW it's from with the index in the labels section. That makes me think that it wouldn't be useful, but someone else might think otherwise.

The name is still used on the user side of things for referencing a filesystem with the environment variables. For example: `DW_JOB_coolfs`